packages/channel-provider/src/types.ts

```diff
@@ -28,13 +28,15 @@ import {
 
 export interface JsonRpcRequest<MethodName = string, RequestParams = any> {
   id?: number;
+  time?: number;
```

Comment:
I don't think this follows the official JSON-RPC spec?

Do we need a `time` property?

Reply:
I don't know. The `request.id` was set as `Date.time()`, so I thought I should keep the time in the request.

I have no reason other than that for keeping it, so I'll remove it.